platform-specific/dtls_prng_win.c

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
+ *
+ * Copyright (c) 2022 Olaf Bergmann (TZI) and others.
```

Comment:
The common practice is to put your name here, if you created the file.
Other projects seems to use " Contributors to the Eclipse Foundation", but for now, I think, your name will be the right one.

Reply:
Thank you, I adjusted the copyright statement in the latest fixup commit.